src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs

```diff
@@ -294,16 +294,11 @@ public void ILLink_verify_analysis_warnings_hello_world_app(string targetFramewo
 
             // Please keep list below sorted and de-duplicated
             List<string> expectedOutput = new List<string> () {
-                    "ILLink : Trim analysis warning IL2026: System.Resources.ResourceReader.InitializeBinaryFormatter(",
-                    "ILLink : Trim analysis warning IL2057: System.Resources.ManifestBasedResourceGroveler.CreateResourceSet(Stream,Assembly",
-                    "ILLink : Trim analysis warning IL2057: System.Resources.ResourceReader.FindType(Int32",
-                    "ILLink : Trim analysis warning IL2060: System.Resources.ResourceReader.InitializeBinaryFormatter(",
-                    "ILLink : Trim analysis warning IL2070: System.Diagnostics.Tracing.NullableTypeInfo.NullableTypeInfo(Type,List<Type>",
-                    "ILLink : Trim analysis warning IL2070: System.Diagnostics.Tracing.TypeAnalysis.TypeAnalysis(Type,EventDataAttribute,List<Type>",
                     "ILLink : Trim analysis warning IL2072: System.Diagnostics.Tracing.EventSource.EnsureDescriptorsInitialized(",
+                    "ILLink : Trim analysis warning IL2026: System.Resources.ManifestBasedResourceGroveler.CreateResourceSet(Stream,Assembly",
```

Comment:
FYI: @tlakollo @sbomer I'm updating the list to remove the System.Resources warnings except one. The reason why we need to keep this one is due to even though we are now by defualt trimming all System.Resources unsafe code when publishing trimmed, this test has the trimMode set to copyifused, so it is not using the substitution file to trim the unsafe use in ResourceManager, hence the warning is expected.

Reply:
This test is sort of odd - what good does the ILLink warnings provide when you are trimming with `copyused`?

Reply:
I think the test serves a purpose by reflecting what the warnings would look like if the default trimming mode is used. At the same time, the fact that the TrimMode can affect the number of warnings I guess it's important to test that too. Given that we no longer expect the warnings to change that much maybe I can create a second test to verify the warnings in the link trim mode.